userspace/libsinsp/sinsp.h

```diff
@@ -570,13 +570,21 @@ class SINSP_PUBLIC sinsp : public capture_stats_source
 	sinsp_evt::param_fmt get_buffer_format();
 
 	/*!
-	  \brief Returns true if the current capture is offline
+	  \brief Returns true if the current capture is happening from a scap file
 	*/
 	inline bool is_capture()
 	{
 		return m_mode == SCAP_MODE_CAPTURE;
 	}
 
+	/*!
+	  \brief Returns true if the current capture is offline
+	*/
+	inline bool is_offline()
+	{
+		return m_mode == SCAP_MODE_CAPTURE || m_mode == SCAP_MODE_TEST;
```

Comment:
[nitpick] So we don't have to update this in 2 places if it ever changes (highly unlikely, I know)
```suggestion
		return is_capture() || m_mode == SCAP_MODE_TEST;
```

Reply:
Agree with Mauro! Great catch :D